docs/lc_call_slips/testing_on_staging.md

```diff
@@ -0,0 +1,21 @@
+### Testing LC Call Slips on staging
+
+One convenient way to try your changes on
+production-like data is to run the process on
+staging:
+
+1. Deploy main to staging.
+1. SSH into one of the staging boxes and run `cd /opt/lib-jobs/current && bundle exec rake lib_jobs:process_newly_cataloged_records`
+1. [Open mailcatcher](../../README.md#staging-mail-catcher).
+1. Download one of the CSVs that you'd like to compare.
+1. Repeat the above steps, but with your branch deployed to staging.
+1. Compare the diff between the two CSVs to see
+if it is what you expect.  Note that the order
+of rows may change, and this doesn't indicate
+a problem.
+1. You may also choose to run `bundle exec rake lib_jobs:create_csv_for_selector_comparison`, which
+generates a CSV of all possible records from OCLC,
+including ones that are typically filtered out like
```

Comment:
It actually filters out the ones that don't meet the criteria for the `generally_relevant?` method. I was just saying that it used to be that the `generally_relevant?` method was where we excluded based on publication location, but now that decision is made in the `relevant_to_selector?` method. 

So, it should exclude juvenile, audiobooks, etc., but will include US / UK / Canada publications.